### PR TITLE
added unit tests and integration test

### PR DESCRIPTION
**Pull request by smileyenot983**
Yes, i need all this comments

### CODE REVIEW

File: integration_test/config/integration_control.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+spider_robot:
+  # Publish all joint states -----------------------------------
+  joint_state_controller:
+    type: joint_state_controller/JointStateController
```
Comment (l1va) [inline integration_test/config/integration_control.yaml]: why did you test in a separate package? what do you test then?

File: integration_test/scripts/publisher.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+	rospy.init_node('talker', anonymous=True)
+	rate = rospy.Rate(50)
+
+
+	while not rospy.is_shutdown():
+
+
+		position111 = 0.2
```
Comment (l1va) [inline integration_test/scripts/publisher.py]: why not to move out of while cycle these constant?

File: integration_test/scripts/subscriber.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+	rospy.spin()
+
+
+
+
+def check_position111(msg):
+	if abs(msg.set_point-msg.process_value) < 0.01:
+		print('joint 11_111 passed test')
```
Comment (l1va) [inline integration_test/scripts/subscriber.py]: where are the asserts?

File: unit_tests/test_fk.py [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+           [ 0.        , -0.94388333,  0.33027906,  0.47941724],
+           [ 0.        ,  0.        ,  0.        ,  1.        ]])
+        fk_answer = FK(-4,-5,-1,-math.pi/4,-math.pi/7)
+        self.assertTrue(np.allclose(fk_answer,result))
+        
+
+
+tests = test_FK()
+
+tests.test_FK1_simple()
```
Comment (l1va) [inline unit_tests/test_fk.py]: it can be done without calling all methods manually.

File: integration_test/config/integration_control.yaml (hunk 1 above)
Comment (smileyenot983) [reply integration_test/config/integration_control.yaml]: I test how publisher, that moves my robot's legs is working

File: integration_test/scripts/publisher.py (hunk 2 above)
Comment (smileyenot983) [reply integration_test/scripts/publisher.py]: Nice idea

File: unit_tests/test_fk.py (hunk 4 above)
Comment (smileyenot983) [reply unit_tests/test_fk.py]: It's feature!

File: integration_test/scripts/subscriber.py (hunk 3 above)
Comment (smileyenot983) [reply integration_test/scripts/subscriber.py]: Good question. May be i will add them one day.

**Pull request merged**